charm/docs/tutorial/getting-started.md

```diff
@@ -75,41 +85,47 @@ This allows uploaded images to be placed on an S3 compatible server instead of
 the local filesystem.
 
 You can configure it with:
-```
+
+```bash
 juju deploy s3-integrator --channel edge
-juju config s3-integrator endpoint="<aws_endpoing_url>" bucket=<bucket_name> path=<path_in_bucket> region=<region> s3-uri-style=<path_or_host>
+juju config s3-integrator endpoint="${AWS_ENDPOINT_URL}" bucket="${AWS_BUCKET}" path=/ region="${AWS_REGION}" s3-uri-style="${AWS_URI_STYLE}"
 juju wait-for application s3-integrator --query='name=="s3-integrator" && (status=="active" || status=="blocked")'
-juju run s3-integrator/leader sync-s3-credentials access-key=<aws_access_key_id> secret-key=<aws_secret_access_key>
+juju run s3-integrator/leader sync-s3-credentials access-key="${AWS_ACCESS_KEY_ID}" secret-key="${AWS_SECRET_ACCESS_KEY}"
 juju integrate s3-integrator netbox
 ```
 
 See the [s3-integrator charmhub page](https://charmhub.io/s3-integrator) for more information.
 
-### Deploy traefik-k8s 
+### Deploy traefik-k8s
 
 You need to enable MetalLb if using MicroK8s. See the [traefik-k8s charmhub page](https://charmhub.io/traefik-k8s) for more information.
 
 With the next example, you can configure Traefik using path mode routing:
-```
+
+```bash
 juju deploy traefik-k8s --channel edge --trust
-juju config traefik-k8s external_hostname=<netbox_hostname>
-juju config traefik-k8s routing_mode=path
+juju config traefik-k8s external_hostname=${EXTERNAL_HOSTNAME}
+juju config traefik-k8s routing_mode=${ROUTING_MODE}
 juju integrate traefik-k8s netbox
 ```
 
 If the host `netbox_hostname` can be resolved to the correct IP (the load balancer IP),
 you should be able to browse NetBox in the url http://netbox_hostname/netbox-tutorial-netbox
 
 You can check the proxied endpoints with the command:
-```
+
+```bash
 juju run traefik-k8s/0 show-proxied-endpoints --format=yaml
+# [docs:traefik-show-endpoints-end]
 ```
 
 ## Create superuser
+
 To be able to login to NetBox, you can create a super user with the next command:
-```
-juju run netbox/0 create-superuser username=<admin_username> email=<admin_email>
+
+```bash
+juju run netbox/0 create-superuser username=admin email=admin@example.com
 ```
 
-Congratulations, With the username created and the password provided in the response, 
+Congratulations, With the username created and the password provided in the response,
 you have now full access to your own NetBox!
```

Comment:
Two comments here:
1. Does it make sense to instruct the user to visit http://netbox_hostname/netbox-tutorial-netbox and check that they have full access?
2. After this section, I recommend adding a `## Clean up the environment` section where the user destroys the Juju model and deletes the Multipass VM (if they used Multipass)

Reply:
1. What would you suggest instead?

Reply:
After the "Congratulations" sentence, I would add one more sentence instructing the user to visit http://netbox_hostname/netbox-tutorial-netbox and log in.